package.json

```diff
@@ -952,6 +952,11 @@
 				"title": "%command.issue.createIssueFromClipboard.title%",
 				"category": "%command.issues.category%"
 			},
+			{
+				"command": "pr.copyPrNumber",
+				"title": "%command.pr.copyPrNumber.title%",
+				"category": "%command.issues.category%"
```

Comment:
Should it be in `%command.pull.request.category%` instead? If so, same should be for `pr.copyVscodeDevPrLink` I think.